docs/access/client.md

```diff
@@ -0,0 +1,194 @@
+  is specified via `CVMFS_HTTP_PROXY`.
+
+As an alternative to defining `CVMFS_CLIENT_PROFILE`, you can also set `CVMFS_HTTP_PROXY` to `DIRECT` to specify
+that no proxy server should be used by CernVM-FS:
+
+``` { .ini .copy }
+CVMFS_HTTP_PROXY="DIRECT"
+```
+
+We will get back to `CVMFS_HTTP_PROXY` later when [setting up a proxy server](proxy.md).
+
+### Quota limit for client cache
+
+The `CVMFS_QUOTA_LIMIT` configuration setting specifies the maximum size of the CernVM-FS client cache (in MBs).
+
+In the example above, we specify that no more than ~10GB should be used for the client cache.
+
+Once the quota limit is reached, CernVM-FS will automatically remove files from the cache according
+to the Least Recently Used (LRU) policy, until half of the maximum cache size has been freed.
+
+The location of the cache directory can be controlled by `CVMFS_CACHE_BASE` if needed (default: `/var/run/cvmfs`),
+but **must** be a on a *local* file system of the client, not a network file system that can be modified by multiple
+hosts.
+
+For more information on cache-related configuration settings,
+[see the CernVM-FS documentation](https://cvmfs.readthedocs.io/en/stable/cpt-configure.html#cache-settings).
+
+
+## Completing the client setup
+
+To complete the setup of the CernVM-FS client component,
+a`cvmfs` user account and group must be added to the system,
+and the `/cvmfs` and `/var/lib/cvmfs` directories must be created.
+
+In addition, it is recommended to update the `autofs` configuration
+to enable auto-mounting of CernVM-FS repositories *(optional)*.
+
+All these actions can be performed in one go by running the following command:
+
+``` { .bash .copy }
+sudo cvmfs_config setup
+```
+
+Additional options can be passed to this command to disable some of the actions,
+like `nouser` to not create the `cvmfs` user and group, or `noautofs` not
+update the `autofs` configuration.
+
+??? note "Impact of not updating `autofs` configuration *(click to expand)*"
+
+    If you prefer not to use `autofs`, you will either need to:
+
+    * Manually mount the CernVM-FS repositories you want to use, for example:
+      ``` { .bash .copy }
+      sudo mount -t cvmfs software.eessi.io /cvmfs/software.eessi.io
+      ```
+
+    * or update `/etc/fstab` to ensure that the CernVM-FS repositories
+      are mounted at boot time.
+
+    For more information on mounting repositories,
+    [see the CernVM-FS documentation](https://cvmfs.readthedocs.io/en/stable/cpt-configure.html#mounting).
+
+## Checking client setup
+
+To ensure that the setup of the CernVM-FS client component is valid, you can run:
+
+``` { .bash .copy }
+sudo cvmfs_config chksetup
+```
+
+You should see `OK` as output of this command.
+
+## Default repositories
+
+The default configuration of CernVM-FS, provided by the `cvmfs-config-default`
+packages, provides the public keys and configuration for a number of
```

Comment:
```suggestion
The default configuration of CernVM-FS, provided by the `cvmfs-config-default`
package, provides the public keys and configuration for a number of
```

Haven't really checked if this is mentioned anywhere, but should we say something about configuration packages (and what you get by default, or how you can change it to `cvmfs-config-none`)?

Reply:
What you get by default is mentioned here (`cvmfs-config-default` is pulled in when you install CVMFS client).

Configuration packages are just one way to add stuff into `/etc/cvmfs`, they're not really special beyond that.

I'll take contributions on `cvmfs-config-none`, doesn't seem very relevant to me in this context, or follow-up in a subsequent PR, it shouldn't block this one imho.